client/my-sites/site-settings/disconnect-site/disconnect-follow-up.jsx

```diff
@@ -0,0 +1,162 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import Gridicon from 'gridicons';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import SelectDropdown from 'components/select-dropdown';
+import { isFreeJetpackPlan, isPersonal } from 'lib/products-values';
+import { isJetpackSite } from 'state/sites/selectors';
+import { getSelectedSite, getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+
+class DisconnectFollowUp extends Component {
+	state = {
+		reasonSelected: 'Backups',
+	};
+
+	logReasonFollowUp = option => {
+		this.setState( {
+			reasonSelected: option.value,
+		} );
+	};
+
+	setOptionsPaidPlan = () => {
+		const { site } = this.props;
+
+		let options = [];
+		options = [
+			{ value: 'Backups', label: 'Backups' },
+			{ value: 'Antispam', label: 'Antispam' },
+			{ value: 'Stats', label: 'Stats' },
+			{ value: 'Publicize', label: 'Publicize' },
+			{ value: 'Subscriptions', label: 'Subscriptions' },
+			{ value: 'Other', label: 'Other' },
+		];
+		if ( ! isPersonal( site.plan ) ) {
+			options.push( { value: 'Security Scanning', label: 'Security Scanning' } );
+		}
+
+		return options;
+	};
+
+	getFollowUpQuestion = optionValue => {
+		const { site, translate } = this.props;
+
+		let textShareWhat = '';
+
+		switch ( optionValue ) {
+			case 'tooHard':
+				if ( isFreeJetpackPlan( site.plan ) ) {
+					textShareWhat = translate( 'Can we help ? ' );
+				} else {
+					textShareWhat = translate( 'Which feature or service caused' + ' you problems?' );
+				}
+				break;
+
+			case 'didNotInclude':
+				textShareWhat = translate( 'Which feature where you looking for? ' );
+				break;
+
+			case 'onlyNeedFree':
+				textShareWhat = translate( 'Would you like to downgrade your plan?' );
+				break;
+		}
+		return textShareWhat;
+	};
+
+	redirectToDisconnect() {
+		// placeholder for action
+		// use redirect-to HoC
+	}
+
+	redirectToPlans() {
+		// placeholder for action
+		// use redirect-to HoC
+	}
+
+	renderFollowUp( optionValue ) {
+		const { site, translate } = this.props;
+
+		switch ( optionValue ) {
+			case 'tooHard':
+				if ( ! isFreeJetpackPlan( site.plan ) ) {
+					const options = this.setOptionsPaidPlan( optionValue );
+					return (
+						<SelectDropdown
+							onSelect={ this.logReasonFollowUp }
+							options={ options }
+							selectedText={ optionValue.value }
+							initialSelected={ 'Backups' }
+						/>
+					);
+				}
+				if ( isFreeJetpackPlan( site.plan ) ) {
+					return (
+						<h1>
+							{ translate( 'Feel free to {{a}}get in touch{{/a}} with our Happiness Engineers.', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **22** times:
`translate( 'No problem! Feel free to {{a}}get in touch{{/a}} with our Happiness Engineers.' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Reply:
split translatable string is fixed